boltdesignsystem/bolt
# Clean and flat all icon in bolt

## Jira

http://vjira2:8080/browse/BDS-2023

## Summary

Clean and flat all icons in the bolt. 

## How to test

Run bolt locally and check if icons rendered correctly. Try to change color of the icon to different one and check if working eg.
`<bolt-icon style="color: #7ACDCD; --bolt-theme-icon: #7ACDCD;" name="face-sad" size="xlarge"></bolt-icon>`
Please check that on various of browsers.
